kernel/generic/include/context.h

```diff
@@ -44,7 +44,7 @@
 		(ctx)->sp = ((uintptr_t) (stack)) + (size) - SP_DELTA; \
 	} while (0)
 
-extern int context_save_arch(context_t *ctx) __attribute__((returns_twice));
+extern errno_t context_save_arch(context_t *ctx) __attribute__((returns_twice));
```

Comment:
This is again rather true (the context save case) / false (the context restore case) case, I don't think context_save_arch can fail.